tests/test_session.py

```diff
@@ -468,3 +469,15 @@ def test_getitem2():
     response = Response(message="I am fine.")
     session = Session(messages=[prompt, response, response, response], title="session")
     assert session[:] == session.messages
+
```

Comment:
Please add additional test cases to improve coverage
You can refer to: https://app.codecov.io/gh/openscilab/memor/pull/90 for guidance

Reply:
added.